lib/cdo/contact_rollups/v2/contact_rollups.rb

```diff
@@ -10,8 +10,10 @@ def self.build_contact_rollups(log_collector)
     log_collector.time!('ContactRollupsComparison.delete_all') {ContactRollupsComparison.delete_all}
     log_collector.time!('ContactRollupsComparison.compare_processed_data') {ContactRollupsComparison.compile_processed_data}
 
-    log_collector.time!('ContactRollupsPardotMemory.add_and_update_pardot_ids') {ContactRollupsPardotMemory.add_and_update_pardot_ids}
+    log_collector.time!('ContactRollupsPardotMemory.add_and_update_pardot_ids') {ContactRollupsPardotMemory.add_and_update_pardot_ids(82_574_936)}
```

Comment:
should this id be here? If it should can you add a comment as to why?

Reply:
Forgotten alternate approach to problem described above ^^^, removed!